source/aggregation-pipeline-builder.txt

```diff
@@ -228,11 +240,9 @@ your chosen view mode:
             To add more aggregation stages to your pipeline, repeat steps 3 
             and 4 for each additional stage. 
             
-            .. note:: 
-               
-               By default, the Stage Wizard adds each new stage to the end of 
-               your pipeline. You can change the order of pipeline stages by 
-               dragging the header of each stage card. 
+            By default, the Stage Wizard adds each new stage to the end of 
```

Comment:
`By default` here refers to clicking a card in the sidebar. However, users can also drag a card and drop it anywhere in between stage and in that case its not the last one.

Reply:
Ah noted — will clarify this ability across both.

Reply:
Adding this to behavior update under step 3, telling users that they can click a card to add it to the end or drag it to their preferred position.